src/Definition/GraphQLServices.php

```diff
@@ -81,4 +82,12 @@ public function getType(string $typeName): ?Type
     {
         return $this->types->resolve($typeName);
     }
+
+    /**
+     * Creates an instance of InputValidator
+     */
+    public function createInputValidator(array $resolverArgs): InputValidator
+    {
+        return $this->services['input_validator_factory']->create($resolverArgs);
```

Comment:
Also for this method args or simply create an object ResolverArgs because we loose al types hints with array.